app/src/bcsc-theme/features/pairing/components/PairingCodeTextInput.tsx

```diff
@@ -25,23 +25,20 @@ const PairingCodeTextInput: React.FC<Props> = ({ handleChangeCode, ...textInputP
     // Clean and normalize the input (uppercase and remove spaces)
     const normalized = text.toUpperCase().replace(/\s/g, '')
 
-    // Check if user is deleting characters by comparing length with previous value
-    const isDeleting = normalized.length < value.replace(/\s/g, '').length
+    // Limit to 6 characters max
+    const limitedNormalized = normalized.slice(0, 6)
```

Comment:
These might be magic numbers? Could they be defined as constants like `const maxCharacterCount = 6`?

Reply:
done